### ISSUE TITLE
crash 

### ISSUE THREAD

**Issue by Haithameng84** [openatv/enigma2#2941]
Attached crash can b[crash for PositionGauge](https://www.mediafire.com/file/9i5adflsotzvef7/20230611-004721-enigma2-crash.zip/file) e treated

**Comment by jbleyel** [comment]
What screen triggers this crash?
This looks like a plugin issue.

**Comment by Haithameng84** [comment]
here ###👇🏻
00:47:21.9712   File "/usr/lib/enigma2/python/Components/Renderer/PositionGauge.py", line 25, in changed
00:47:21.9716   File "/usr/lib/enigma2/python/Components/Renderer/PositionGauge.py", line 46, in setLength
00:47:21.9719 TypeError: in method 'ePositionGauge_setLength', argument 2 of type 'pts_t'
It's in a file PositionGauge

**Comment by jbleyel** [comment]
The value for setLength is wrong. This value comes from a Converter.

**Comment by jbleyel** [comment]
The root cause is this function:

https://github.com/oe-alliance/enigma2-plugins/blob/master/eventposition/src/EventPosition.py#L37-L47

The return must be an integer.

**Comment by 4l3x2k** [comment]
@Haithameng84 
Is it related to DVD Playback with EMC?

**Comment by Haithameng84** [comment]
so what do I do

**Comment by jbleyel** [comment]
Please copy this file : 
[EventPosition.py.txt](https://github.com/openatv/enigma2/files/11731930/EventPosition.py.txt)

to your box in this folder

`/enigma2/python/Components/Converter`

and then rename to EventPosition.py

**Comment by jbleyel** [comment]
You have forgotten to include some important extra info.
A) you are using ServiceApp
B) you are streaming an IPTV channel

This crash needs more investigation.

I can only fix this if I can reproduce the crash.

**Comment by 4l3x2k** [comment]
@Haithameng84 
What type of media are you trying to play? Is it a recording or local media file?
Are there any .cuts involved that ask you to rusume a position in a movie?
Could you enable and provide the debuglog so we can see what leads to the crash?

**Comment by Haithameng84** [comment]
With ServiceApp and IPTV channels, the problem is solved, but with some plugins there is a crash.  thanks bro

**Comment by Haithameng84** [comment]
Yes, it is from the serviceapp plugin. Any plugin that has drivers gets crash.

**Comment by 4l3x2k** [comment]
@Haithameng84 
So this issue can be closed?

If not, tell us exactly which plugin crash.

**Comment by Haithameng84** [comment]
Any plugin that includes video and audio players for iptv, plugins, or even recordings

**Comment by jbleyel** [comment]
We need a step by step description howto reproduce.

**Comment by Haithameng84** [comment]
It has to do with serviceapp, when activated it gets crash and any player that interferes with it, such as recordings when triggered from pvr button, also iptv channels,  and any plugin that has times beep.

**Comment by jbleyel** [comment]
Please reopen if there is still an issue.
